content/help/ops/traffic-management/troubleshooting/index.md

```diff
@@ -4,8 +4,38 @@ description: Describes tools and techniques that can be used to root cause netwo
 weight: 5
 ---
 
-* Migrate content from old troubleshooting guide here
+The following is a general high-level trouble shooting guide for networking issues in Kubernetes environment.
 
-* Provide a few general procedures that should be followed to isolate
+## Pod reachability issues
 
-* Describe high level isolation steps and things to check.
+Usually, the first thing you want to check is whether or not the pod associated with your app has been allocated a valid IP address. This can be done by examining the pod status with `kubectl get pods -o wide`. In addition, check whether or not the service associated with the app has been properly created with a service IP. Assuming the pod is up and running and an IP is allocated, ping the pod IP and make sure it can be pinged. Otherwise, a few things that may be tried to isolate the issue:
+
+* Where is it that you’d like to access the app: from inside the same cluster or outside the cluster. If it’s outside the cluster, find out if port-forward is needed, if the right service type is configured for the service associated with the app, etc. If port-forward is needed, use the `kubectl port-forward` command to configure it.
+* Ping the node where the pod is running.
+* Check route table to make sure that there is a route to the pod IP: `ip route` or `route -n`.
+* Check to make sure that the return packet path is also properly configured. This involves checking the route tables, iptables, etc, in the node where the pod is running.
+* Check iptables to make sure that packets are not dropped.
```

Comment:
I don't think this is a kubernetes issue that people report. When someone says pod is not reachable after installing istio, its pretty much certain that some istio setup is causing the problem. The steps described here do not address that area.